src/views/Resource.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <main v-if="this.error">
+    <h1>Resource Instance not found.</h1>
+  </main>
+  <main v-else class="type">
+    <ViewHeader :title="this.resourceInstance.id" :backLink="{ link: '/resources' }" />
+
+    <ListSection title="Resource Type" class="preview-container" :list="resourceTypeList" />
+
+    <div class="row">
+      <ListSection title="Attributes" :list="resourceInstanceAttributeList" />
+      <ListSection title="Actions" :list="resourceInstanceActionsList" />
+    </div>
+
+  </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import {
+  ResourceInstances,
+  ResourceInstance,
+  ResourceInstanceNullObject,
+  ResourceTypes,
+} from '@/apis/rembrandt/rembrandt';
+import { ListEntry } from '@/components/Li.vue';
+import ListSection from '@/components/ListSection.vue';
+import ViewHeader from '@/components/ViewHeader.vue';
+import Utils from '@/utils/Utils';
+
+@Component({
+  components: {
+    ListSection,
+    ViewHeader,
+  },
+})
+export default class Resource extends Vue {
+  // region public static methods
+  // endregion
+
+  // region private static methods
+  // endregion
+
+  // region public members
+  public resourceInstance: ResourceInstance;
+  public error: string = '';
+
+  public get resourceTypeList(): ListEntry[] {
+    return Utils.resourceTypesToList([ this.resourceInstance.resourceType ]);
+  }
+
+  public get resourceInstanceAttributeList(): ListEntry[] {
+    return Utils.resourceInstanceAttributesToList(this.resourceInstance.attributes);
+  }
+
+  public get resourceInstanceActionsList(): ListEntry[] {
+    const resourceInstanceActions = [];
+
+    resourceInstanceActions.push(
+      {
+        id: '1',
+        firstValue: 'Create new Instance',
```

Comment:
I don't know if we need this button. if we decide to keep it, we should rename it to "Create new resource instance of this type" so we can call the same followup as if you click the button in the resource view.